### ISSUE TITLE
Typescript definitions are not part of the npm package

### ISSUE THREAD

**Issue by janwo** [manuelbieh/Geolib#140]
As the title says, the definitions are missing. Is it possible to add them shortly?

**Comment by jppgks** [comment]
@manuelbieh Could you release from the current master which includes the type file?

**Comment by chetbox** [comment]
Our workaround for now is to use point to a commit directly on GitHub.

```json
  "dependencies": {
    "geolib": "git://github.com/manuelbieh/geolib.git#cc6310b47f2bbdee1b0c0036a97d66afa29676e9"
  }
```

**Comment by manuelbieh** [comment]
Should be included in 2.0.23. Sorry for the long delay. I'm pretty much over capacity with my freelance projects at the moment.
